templates/helpers/operation.js

```diff
@@ -1,6 +1,26 @@
 const _ = require('lodash');
 
-const getBodyModelNameInCamelCase = operation => {
+const MODELS_SHOULD_NOT_PROCESS = ['object', 'string', 'undefined'];
+
+// BEGIN work around spec mismatches and upstream parsing incosistencies
+const RESTRICTED_MODEL_PROPERTY_OVERRIDES = {
```

Comment:
This is good work dealing with the exceptions, but I'm worried the next person to touch this may not follow, particularly since the templates say to set the property UNLESS it is an override (the opposite of what I'd expect an override to do).  

Could you drop in a 1-2 line comment clarifying the logic here?

Reply:
Sure - I was hoping that could be read as 'add a property unless it's an illegal override', but that didn't work out so well 😅